src/components/Features/Search.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <input
```

Comment:
Considering that the class name and placeholder values are dependent on user configurations, consider making this component only visible if the user passes the needed configuration, otherwise the search component will always be shown regardless if the user has passed configuration for it to show.

Reply:
I applied conditional rendering to the Search component. The component will now only be rendered if config object is set.

Reply:
Alright :+1: